Modules/_sqlite/connection.c

```diff
@@ -33,7 +33,9 @@
 #define ACTION_FINALIZE 1
 #define ACTION_RESET 2
 
-#if SQLITE_VERSION_NUMBER >= 3014000
+#if SQLITE_VERSION_NUMBER >= 3014000 || \
+    defined(__APPLE__) && defined(__MACH__) \
+    && __MAC_OS_X_VERSION_MIN_REQUIRED >= __MAC_10_12
```

Comment:
This will fail if you're compiling against SQLite pre 3.14 on macOS. One cannot assume that all macOS builds uses the SQLite 3 that's shipped with the OS.

Reply:
Maybe it should check in the configure; and set the `HAVE_TRACE_V2` there?